### ISSUE TITLE
bettingRound() implementation

### ISSUE THREAD

**Issue by derderdunichbist** [derderdunichbist/poker#5]
Complete the implementation of the bettingRound() method

Missing implementation: 

1. Detecting an `allin raise`. Then according to the situation handling this special case for the upcoming rounds

**Comment by SebastianZeeb** [comment]
Updated on 22_02_16. Implementation is mostly done. Folding,Checking,Calling and Raising work properly. Betting can be done until the very last round, a winner will be determined if there is only one player left in the ongoing betting-round. If there is a showdown, `identifyHands()` will be called. 

The only thing missing is a way to handle an allin-situation, which skips bettings for all players that are committed by their entire stack, as well as it reveals their cards for the board. 

Note: There can be several layers of allins on top of each other, which will deny the revelation of cards as well as the possibility for further betting. 

In other words:

1. Players that are "all-in" can not bet any more and are simply skipped
2. Even after a player is "all-in" there can still be betting going on. Allin is simply the new `toCallAmount ` for the other players. 
3. In the case that a player is allin but there have been several more raises on top of that amount, the allin-player either loses his stack or can only win a multiple of his `bettedAmount` if he holds the strongest hand. In that case, the rest of the pot is being split according to the rules between the remaining players (note that there can be several more betting-layers here too!)
